altsrc/flag.go

```diff
@@ -231,3 +255,19 @@ func isEnvVarSet(envVars []string) bool {
 func float64ToString(f float64) string {
 	return fmt.Sprintf("%v", f)
 }
+
+func stringToPath(context *cli.Context, path string) (string, error) {
+	inputSourcePath := context.String("load")
```

Comment:
I see, yes, it'll be difficult to access the path of the loaded file from here.

One approach could be to add another method to `InputSourceContext` that returns the filepath of the source; this value would then be passed in by the YAML and TOML loaders to the constructed MapInputSource.

Reply:
Yeah, I was thinking in the same direction. I'll do that. Cheers!

Reply:
Done.